src/dictionary.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     skills: ['skills', 'Skills & Expertise', 'technology', 'technologies'],
     languages: ['languages'],
     courses: ['courses'],
-    projects: ['projects'],
+    projects: ['projects','personal projects'],
```

Comment:
please, add a whitespace after the comma

Reply:
I'll make the necessary changes